tests/test_fernet.py

```diff
@@ -56,19 +65,24 @@ def test_generate(self, secret, now, iv, src, token, backend):
         ("secret", "now", "src", "ttl_sec", "token"),
         "verify.json",
     )
+    @pytest.mark.parametrize("typ", (str, bytes))
```

Comment:
I'd prefer if instead of using `parameterize` for types we just wrote a second test case that covered this variant, e.g. as you did in `test_extract_timestamp`.

Reply:
🤷 cando.